src/main/java/service/UserService.java

```diff
@@ -0,0 +1,37 @@
+package service;
+
+import exception.LoginCommandException;
+import exception.UserNotFoundException;
+import model.User;
+import model.UserStatus;
+import repository.UserRepository;
+import repository.UserSession;
+
+public class UserService {
+
+    public void login(String username, String password) throws LoginCommandException {
+
+        try {
+            User foundUser = UserRepository.findUserByUsernameAndPassword(username, password);
+            if (isActive(foundUser)) {
+                UserSession.currentUser = foundUser;
+            }
+        } catch (UserNotFoundException e) {
+            throw new LoginCommandException("you are blocked");
+        }
+
+    }
+
+    public boolean isUserExists(String userName) {
```

Comment:
You can call repository method inside for finding based by username. Instead of getUsers